R/cassette_class.R

```diff
@@ -441,32 +422,13 @@ Cassette <- R6::R6Class(
       self$record == "all"
     },
 
-    #' @description Get character string of entire cassette; bytes is a misnomer
-    #' @return character
-    raw_cassette_bytes = function() {
-      file <- self$file()
-      if (is.null(file)) return("")
-      tmp <- readLines(file) %||% ""
-      paste0(tmp, collapse = "")
-    },
-
-    #' @description get http interactions from the cassette via the serializer
-    #' @return list
-    deserialized_hash = function() {
-      tmp <- self$serializer$deserialize(self)
-      if (inherits(tmp, "list")) {
```

Comment:
It's the responsibility of the deserializer to error if there's a problem, and there are lots of potential problems that we don't currently error for.

Reply:
i like it